draft-packages/modal/KaizenDraft/Modal/Primitives/ModalFooter.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 
-import { Button } from "@kaizen/component-library"
+import { Button } from "@kaizen/draft-button"
```

Comment:
Does this mean the primary button will now be Cluny? It's hard to tell just looking at the code.

Reply:
descoped

Reply:
Rescoped after discussing with Max + Doug from the a11y capture project